src/ui/components/JEXLDebuggerPage.tsx

```diff
@@ -57,6 +76,41 @@ const JEXLDebuggerPage: FC = () => {
     }
   }, [jexlExpression, clientContext, addToast]);
 
+  const parseAndSetContext = useMemo(() => {
+    return debounce((key: string, value: string) => {
+      try {
+        const parsedValue = JSON.parse(value) as ContextValue;
+        setClientContext((prevContext) => ({
+          ...prevContext,
+          [key]: parsedValue,
+        }));
+      } catch (error) {
+        addToast({
+          message: `Error changing context: ${(error as Error).message ?? String(error)}`,
+          variant: "danger",
+        });
+      }
+    }, 1000);
+  }, [addToast, setClientContext]);
+
+  const handleContextChange = useCallback(
+    (key: string, value: string, isText: boolean) => {
+      if (isText) {
+        setClientContext((prevContext) => ({
```

Comment:
We should keep the context and user overrides separate, so that we keep type information of the original types (for rendering fields and parsing values from the formData).

We can then provide `{...clientContext, ...overrides}` to the eval function.

We would then also know if values had changed, so we could also add reset buttons for fields (optional)